website/projects/map4sci/src/app/app.component.html

```diff
@@ -7,11 +7,53 @@
 </select>
 
 <div class="map-wrapper">
-  <m4s-map
-    *ngIf="displayMap"
-    [edgeFeatures]="dataset.edges"
-    [nodeFeatures]="dataset.nodes"
-    [clusterFeatures]="dataset.cluster"
-    [boundaryFeatures]="dataset.boundary">
-  </m4s-map>
+  <ng-container *ngIf="displayMap">
+
+    <!--
```

Comment:
@bherr2 I don't know if it's just late or I'm spacing it but how can I make it so that the undefined being passed in won't overwrite the default I'm setting without having to do this messy ngIf else block?